### PR TITLE
Enable prow jobs for release-4.10 branch for Power VS

### PR DESCRIPTION
**Pull request by Karthik-K-N**
This PR contains the changes to enable prow jobs for release-4.10 branch for Power VS

### CODE REVIEW

File: ci-operator/jobs/openshift/machine-api-provider-aws/openshift-machine-api-provider-aws-release-4.10-presubmits.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ presubmits:
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
   - agent: kubernetes
     always_run: false
-    branches:
+    branches:govet
```
Comment (mkumatag) [inline ci-operator/jobs/openshift/machine-api-provider-aws/openshift-machine-api-provider-aws-release-4.10-presubmits.yaml:78]: why this change? seems like this change is creating CI failure!

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openshift/machine-api-provider-aws/openshift-machine-api-provider-aws-release-4.10-presubmits.yaml (hunk 1 above)
Comment (Karthik-K-N) [reply ci-operator/jobs/openshift/machine-api-provider-aws/openshift-machine-api-provider-aws-release-4.10-presubmits.yaml:78]: Sorry, That was not intended, Reverting it back

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Pull request merged**